tests/docker-compose.yml

```diff
@@ -2,20 +2,40 @@ version: '3.1'
 
 services:
 
-  db:
-    image: postgres:9.0
+  db90:
+    image: postgres:9.0.22
     restart: always
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: example
       POSTGRES_DB: demo
 
+  db91:
+    image: postgres:9.1.24
+    restart: always
+    depends_on:
+      - db90
```

Comment:
```suggestion
```
Rather than starting all test services sequentially (making them depend on each other), we could let them start in parallel and make `nri-postgresql` to depend on the three of them

Reply:
👍